pandas/tests/indexing/test_iloc.py

```diff
@@ -269,6 +269,14 @@ def test_iloc_setitem(self):
         expected = Series([0, 1, 0], index=[4, 5, 6])
         tm.assert_series_equal(s, expected)
 
+        df = pd.DataFrame(data=[[1, 2, 5], [1, 3, 6]], columns=['i', 'j', 'k'])
```

Comment:
make a new test
add the issue number as a comment.

pls test all cases brought up.